add new models: Product, ExpenseCategory

krish300 opened the pull request.

krish300 committed:
add new models: Product, ExpenseCategory

arabindadora commented on backend/api/models.py:
```diff
@@ -1,3 +1,14 @@
 
+
+class ExpenseCategory(models.Model):
+    name = models.CharField(max_length=50)
```
make name unique

arabindadora commented on backend/api/models.py:
```diff
@@ -1,3 +1,14 @@
+
 # Create your models here.
+class Product(models.Model):
+    name = models.CharField(max_length=50)
```
make name unique

arabindadora commented on backend/api/models.py:
```diff
@@ -1,3 +1,14 @@
+class Product(models.Model):
+    name = models.CharField(max_length=50)
+    category = models.ForeignKey(
+        ExpenseCategory, on_delete=models.SET_NULL, null=True)
```
use the recommended way of putting quotes around FK model names.

krish300 committed:
change as per review suggestions

krish300 committed:
fix format

krish300 committed:
Multiple updates to models
- add Bill, Person, EntryCategory, Entry
- add created_ts, last_modified_ts to existing models

krish300 committed:
Chnages as per review

arabindadora reviewed (approved):
LGTM
